.github/workflows/python-ci.yaml

```diff
@@ -9,7 +9,7 @@ env:
   POETRY_VIRTUALENVS_IN_PROJECT: true
 
 jobs:
-  build:
+  Build:
     runs-on: ubuntu-latest
     container: python:3.10-buster
```

Comment:
Let's try with Python 3.11

Reply:
Confirmed successfully installations (local and CI) with `python3.11`. Includes `tensorflow` as well.